src/ineqpy/inequality.py

```diff
@@ -452,3 +453,65 @@ def avg_tax_rate(total_tax, total_base, weights=None, data=None):
     names = ["_".join([t, b]) for t, b in zip(tax_name, base_name)]
     res = pd.Series(res, index=names)
     return res
+
+
+def ratio_top_rest(income, weights=None, data=None, percentage=10.0):
```

Comment:
What do you think about rename `percentage` by `top` ?

Reply:
I'll use top_percentage to make it clearer that it should be a number between 0 and 100